data/core/units/monsters/Roc.cfg

```diff
@@ -6,44 +6,79 @@
     num_traits=2
     undead_variation=gryphon
     image="units/monsters/roc.png"
-    hitpoints=44
+    profile="portraits/monsters/roc.webp"
+    hitpoints=76
     movement_type=fly
     movement=10
     experience=100
-    level=2
+    level=3
     alignment=neutral
     advances_to=null
     {AMLA_DEFAULT}
     cost=36
     usage=scout
-    description= _ "Far from human cities in the great desert mountains, the large birds of prey known as Rocs can sometimes be found rapidly scouring the sands on their hunts. Despite their large size, Rocs tend to be reclusive and are only rarely encountered by other sapient races. Their elusive nature has raised them to near mythological status, particularly among the nomads of the deserts.
+    description= _ "In remote desert mountains, far from any city or settlement, enormous raptors are rumoured to nest in the lofty crags.  Travellers and nomads claim to have witnessed livestock and grown men snatched into the sky without warning by these monsters.  While one should not place much faith in the accounts of the desert roamers, they are less fantastical than the drunken lies of sailors who claim to have witnessed full-rigged ships, attempting to navigate the narrow straights between the sheer cliffs where nothing can grow, sunk by giant eagles dropping large rocks.
```

Comment:
> narrow straights

I think this should be 'narrow straits'. While strait is, by definition, narrow, I read this as intended to convey a picture of a passage perhaps only a dozen or two metres wide (between the sheer cliffs), so I'm okay with leaving narrow there.

@Pentarctagon Another correction needed post-freeze.

Reply:
Could you open an issue that lists anything you find?

Reply:
Planning to do so.